pastis/e2e_simulators/webbpsf_imaging.py

```diff
@@ -166,6 +162,32 @@ def set_up_nircam():
 
     return nircam, ote
 
+def set_up_cgi():
+    """
+    Return a configured instance of the CGI simulator on RST.
+
+    Sets up the Lyot stop and filter from the configfile, turns of science instrument (SI) internal WFE
```

Comment:
"turns of" --> "turns off"

Reply:
Also, please add a sentence that says that the FPM setting will also be read from the configfile (no FPM, or, a specific FPM). E.g. "FPM setting (out of beam = 'OFF', or a specific FPM name) is also read from the configfile."